Lectures/Lecture8_OpenScienceReflections.md

```diff
@@ -1,41 +1,97 @@
-# Open Science for Physicists, Lecture [number] , [main topic]
+# Open Science for Physicists, Lecture 8 , Open Science Reflections
 
-*This is a template file, please make a copy in the ../Lectures director before customizing for a specific lecture*
-
-+ Last updated: DD MM 20YY
++ Last updated: 30 08 2024
 + Lecturer: e.g. @SanliFaez 
 
 #### Contributors: 
 + Erik van Sebille @erikvansebille
-+ Hosein Nemati
++ Tanja Hinderer
 
 ## Development goal
-__why are we covering this topic and how can this lecture help participants progress towards that learning goal __
-
+ To reflect on the key concepts in open science that we have discussed in this course. 
+ To practice storytelling and video production skills in creating a short (scientific) pitch.
```

Comment:
Add 'to be discussed in a symposium-style setting', so that the symposium in the motivation doesn't come out of the blue?